OSDOCS-17079-2: CQA 2.0 for SUPP-1: Remote Health Monitoring (Insight (#114850)
openshift/openshift-docs: modules/insights-operator-manual-upload.adoc

----
$ curl -v -H "User-Agent: insights-operator/one10time200gather184a34f6a168926d93c330 cluster/_<cluster_id>_" -H "Authorization: Bearer _<your_token>_" -F "upload=@_<path_to_archive>_; type=application/vnd.redhat.openshift.periodic+tar" https://console.redhat.com/api/ingress/v1/upload
----

> +

Reviewer: I don't see [this](https://github.com/openshift/openshift-docs/pull/102276/changes#diff-df8adb1bd0e2ca4267d7e91c67dabe826f8d26f0d60ee8854c4d2042c4e434ecR86) as an option. I get though that I think you were trying to keep to the original as much as possible, but I think it might be better to fix this following the guidelines.

Author: :heavy_check_mark:

> Where `_<cluster_id>_` is your cluster ID, `_<your_token>_` is the token from your pull secret, and `_<path_to_archive>_` is the path to the {insights-operator} archive.

Reviewer: ```suggestion
Replace `_<cluster_id>_` with your cluster ID and `_<your_token>_` with the token from your pull secret, and `_<path_to_archive>_` with the path to the {insights-operator} archive.
```

Reviewer: https://github.com/openshift/openshift-docs/pull/102276/changes#diff-a4543a07ef809868d1b552e88f1c5a5b4dbbc99a3ff5c769ec4ad63cd55d0b4dR126

Reviewer: I can also see why you might do 
+

where:
`cluster_id`:: Specifies...
`your_token`:: Specifies...

Author: :heavy_check_mark:
